recipe/run_test.py

```diff
@@ -1,50 +0,0 @@
-#!/usr/bin/env python
```

Comment:
I am not 100% OK removing this test just because we cannot install `xz` on Python 3.5, but once https://github.com/conda-forge/staged-recipes/pull/370 get sorted out we can bring this test back.